(Mock)kickoff updates (#146)
FRCTeam3255/Wiki: Build Season/Day 1.1 - Game Manual Breakdown.md

| 9   | Robot construction (~30pgs)<br>focus on starting config (9.1)                                                                                |          2           | Eli    |
| 6   | Match Play (~10 pgs)                                                                                                                         |          3           | Tayler |

## Manual Breakdown steps

### Reading

!!! tip
    Just read title of evergreen rules

1. Read to group [research questions](#research-questions) as examples of what they should be looking for/thinking about
2. Break into groups
    1. 3 groups, 2 mentors per group
    2. Different manual sections
    3. Each group has Google Slide or Google Docs for notes
3. Within groups
    1. Designate primary scribe
    2. Read each paragraph aloud
    3. Take free form notes on key takeaways

>        1. Copy/paste original rule

Reviewer: ```suggestion
       1. Copy/paste original rule, into scratch notes or slides
```

       2. Copy/paste pictures and tables
       3. Write interpretation of rule
    4. Record any questions about the manual/rules that come up
    5. After reading and taking notes refine into presentation
       1. Rule, then takeaway/interpretation
    6. If time permits, answer [research questions](#research-questions) as a group
       1. Record answers in notes doc

### Presenting

1. Regroup as whole team
2. 2 presenters for each group come up

> 3. Mentors ask 1 [research question](#research-questions) at a time

Reviewer: ```suggestion
3. Have student run through slides, and answer any questions from group
4. Once presentations are done, begin running through research questions with whole team making a master list. Ask 1 [research question](#research-questions) at a time.
```

>     1. Presenters state their answers to questions regardless of group

Reviewer: ```suggestion
    1. Students state their answers to questions regardless of group
```

>     2. Create master list of answers to [research questions](#research-questions)

Reviewer: ```suggestion
    2. Fill out master list of answers to [research questions](#research-questions)
```

    3. Repeat for next question

>     4. Any presenter can add additonal comments to a question, as well as provide their answer if already answered

Reviewer: ```suggestion
```

4. Review master list of analysis questions and fill gaps/come back to anything we skipped
5. Each individual group presents extra notes/questions
    1. Create master list of extra notes
    2. Create master list of questions to ask FIRST Q&A

## Research questions

!!! Important
    When recording answers cite section/subsection/paragraph (e.g. 5.1 paragraph 2) where the information was found

### What are point values?

> - Are points control time based or per quantity based?

Reviewer: ```suggestion
- Are points given based on quantity scored, or time of ownership (ex. 2018), or both?
```

### Zones (boundaries)

- Where are Protected Zones?
  - What are their penalty points?

> - Where are Scoring Zones?

Reviewer: ```suggestion
- Where are Scoring Zones? (Field Zones you can legally score in)
```

### Scoring Elements (aka Game Pieces)

- How many scoring elements can the robot preload?

> - How many scoring elements can the robot hold?

Reviewer: ```suggestion
- How many scoring elements can the robot legally hold?
```
